### DISCUSSION TITLE
Restructure CiviWiki organization to facilitate community stewardship

### DISCUSSION THREAD

**Discussion by brylie** [CiviWiki/OpenCiviWiki#1409] [Ideas]
Currently, the CiviWiki organization has three members:

- @mdwest (Owner)
- @JOO33 (Owner)
- @brylie (Member)

While we are trying to recognize all contributors in our README, it may be more beneficial to invite contributors to join our GitHub organization.

As a first step towards greater community involvement in the CiviWiki project, let's:

1. create one or more groups in the CiviWiki organization for contributors (e.g. Design, Development, Documentation)
2. invite all current and previous contributors to join the CiviWiki organization by adding them to the relevant team

## List of contributors to invite
In the same order as our README:

- @lisbonjoker
- @TomaszKolek 
- @Unit111 
- @Sv3nman 
- @ryan-barrett 
- @WCollins3 
- @delta-plus 
- @GokuMizuno 
- @dborstelmann
- @DCalliet 
- @tractaylor 
- @tomjohnhall 
- @jstrang 
- @danielgrijalva 
- @g1stavo 
- @pscheid2 
- @terrencetuy 
- @JonanOribe 
- @shaunak-gupta 
- @dlarner3194 
- @darcymeyer 
- @Jazriel 
- @therohitramesh 
- @richifuentes
- @alecxvs
- @Shreyas4991
- @SunflowerPKU
- @jgbmattos
- @torabshaikh 
- @CaioSR 
- @umr55766 
- @JDRanpariya 
- @Di-ref 
- @BrianPainter 
- @AnkanDas22
- @gteti 
- @harsha0609 
- @obafemiolorungbon 
- @tberm 
- @PyDrummer

**Comment by brylie** [comment]
@mdwest or @JOO33, would you mind adding me as an Owner in the CiviWiki organization so I can work on the above-mentioned changes?

**Comment by brylie** [comment]
Ping @mdwest and @JOO33

**Comment by AbhijithGanesh** [comment]
I would like to be a part time steward if there is a need of it.

**Comment by brylie** [comment]
@mdwest or @JOO33, would you mind adding me as an Owner on the [CiviWiki organization members page](https://github.com/orgs/CiviWiki/people)?

![image](https://user-images.githubusercontent.com/17307/134664910-68fc9f19-707e-4fa4-9e19-72f127ccd1e5.png)

**Comment by gteti** [comment]
Thank you 😃

**Comment by brylie** [comment]
I propose we create a new organization and move the `OpenCiviWiki` repository to that organization to have complete stewardship control.

**Reply by gorkemarslan** [reply]
Sounds fine to me. 👍 

Just out of curiosity: Having full control over the repository would be better. Is there any drawback to transferring a repository to another organization? I think we won't lose the number of stars, forks, issues, pull requests, etc., right?

What do you think of as an organization name? 

A side note: When CiviWiki and OpenCiviWiki were once created OpenCV library wasn't well known like today. The names of our project and the OpenCV library sound quite similar.

**Reply by brylie** [reply]
Hmm, well, we might consider a different name than CiviWiki. We should open a separate discussion for assessing the project/organization rename.
